google-cloud-beyond_corp/lib/google-cloud-beyond_corp.rb

```diff
@@ -20,3 +20,4 @@
 require "google/cloud/beyond_corp/app_connectors" unless defined? Google::Cloud::BeyondCorp::AppConnectors::VERSION
 require "google/cloud/beyond_corp/app_gateways" unless defined? Google::Cloud::BeyondCorp::AppGateways::VERSION
 require "google/cloud/beyond_corp/client_gateways" unless defined? Google::Cloud::BeyondCorp::ClientGateways::VERSION
+
```

Comment:
Consider doing this in a non-ruby file such as the readme so rubocop doesn't get triggered.

Reply:
Thanks, I fixed it.